src/views/Game/GameDock.tsx

```diff
@@ -325,123 +329,157 @@ export function GameDock({
+                                ? "fa-volume-off"
+                                : volume > 0.5
+                                ? "fa-volume-up"
+                                : "fa-volume-down")
+                        }
+                        onClick={toggleVolume}
+                    />{" "}
+                    <input
+                        type="range"
+                        className="volume-slider"
+                        onChange={setVolume}
+                        value={volume}
+                        min={0}
+                        max={1.0}
+                        step={0.01}
+                    />
+                </a>
+            </Tooltip>
+            <Tooltip tooltipRequired={tooltipRequired} title={_("Zen mode")}>
+                <a onClick={onZenClicked}>
+                    <i className="ogs-zen-mode"></i> {_("Zen mode")}
+                </a>
+            </Tooltip>
 
-            <a>
-                <i
-                    className={
-                        "fa volume-icon " +
-                        (volume === 0
-                            ? "fa-volume-off"
-                            : volume > 0.5
-                            ? "fa-volume-up"
-                            : "fa-volume-down")
-                    }
-                    onClick={toggleVolume}
-                />{" "}
-                <input
-                    type="range"
-                    className="volume-slider"
-                    onChange={setVolume}
-                    value={volume}
-                    min={0}
-                    max={1.0}
-                    step={0.01}
-                />
-            </a>
-
-            <a onClick={onZenClicked}>
-                <i className="ogs-zen-mode"></i> {_("Zen mode")}
-            </a>
-            <a onClick={onCoordinatesClicked}>
-                <i className="ogs-coordinates"></i> {_("Toggle coordinates")}
-            </a>
-            {game && (
-                <a onClick={onAIReviewClicked}>
-                    <i className="fa fa-desktop"></i>{" "}
-                    {ai_review_enabled ? _("Disable AI review") : _("Enable AI review")}
+            <Tooltip tooltipRequired={tooltipRequired} title={_("Toggle coordinates")}>
+                <a onClick={onCoordinatesClicked}>
+                    <i className="ogs-coordinates"></i> {_("Toggle coordinates")}
                 </a>
-            )}
-            <a onClick={showGameInfo}>
-                <i className="fa fa-info"></i> {_("Game information")}
-            </a>
+            </Tooltip>
+
             {game && (
-                <a
-                    onClick={onAnalyzeClicked}
-                    className={phase !== "finished" && goban.isAnalysisDisabled() ? "disabled" : ""}
-                >
-                    <i className="fa fa-sitemap"></i> {_("Analyze game")}
+                <Tooltip tooltipRequired={tooltipRequired} title={_("Toggle AI review")}>
```

Comment:
I might be making this up, but I seem to recall once having "Toggle AI review" as the text here, and it being confusing enough to people that I ended up doing what we do down below which is to be more explicit with `i_review_enabled ? _("Disable AI review") : _("Enable AI review")`